src/actions/app.js

```diff
@@ -1,7 +1,8 @@
 import axios from 'axios';
-
+import { removeDeprecatedFiltersFromState } from '../util/codemirror-utils';
```

Comment:
The name was not created in this PR, and its likability is subjective. It does not effect the functionality, and changing the name won't necessarily improve it. Shouldn't be a blocker.